frontend/packages/network-attachment-definition-plugin/src/constants/index.ts

```diff
@@ -1 +1,66 @@
 export const NET_ATTACH_DEF_HEADER_LABEL = 'Create Network Attachment Definition';
+
+export const ELEMENT_TYPES = {
+  CHECKBOX: 'checkbox',
+  DROPDOWN: 'dropdown',
+  TEXT: 'text',
+  TEXTAREA: 'textarea',
+};
+
+export const networkTypes = {
+  sriov: 'SR-IOV',
+  bridge: 'Linux bridge',
+};
+
+export const networkTypeParams: NetworkTypeParamsList = {
+  sriov: {
+    resourceName: {
+      name: 'Resource Name',
+      values: {},
+      required: true,
+      type: ELEMENT_TYPES.DROPDOWN,
+    },
+    vlanTagNum: {
+      name: 'VLAN Tag Number',
+      hintText: 'Ex: 100',
+      type: ELEMENT_TYPES.TEXT,
+    },
+    ipam: {
+      name: 'IP Address Management',
+      type: ELEMENT_TYPES.TEXTAREA,
+    },
+  },
+  bridge: {
+    bridgeName: {
+      name: 'Bridge Name',
+      required: true,
+      type: ELEMENT_TYPES.TEXT,
+    },
+    vlanTagNum: {
+      name: 'VLAN Tag Number',
+      hintText: 'Ex: 100',
+      type: ELEMENT_TYPES.TEXT,
+    },
+    ipam: {
+      name: 'IP Address Management',
+      type: ELEMENT_TYPES.TEXTAREA,
+    },
+  },
+};
+
+type NetworkTypeParamsList = {
+  [key: string]: NetworkTypeParams;
+};
+
+export type NetworkTypeParams = {
+  [key: string]: NetworkTypeParameter;
+};
+
+export type NetworkTypeParameter = {
+  name: string;
+  required?: boolean;
+  type: string;
+  validation?: (...args: any[]) => string | null;
```

Comment:
cant we have something better than `any` ?

Reply:
I just removed it. None of the current networks have field validation at the moment, so it's not necessary. I'll add better typing if I end up adding validation for parameters later on.